components/SignUpForm.tsx

```diff
@@ -193,35 +217,31 @@ const SignUpForm: React.FC = () => {
                 <div className='rsvp-form'>
                   <label htmlFor='email'>Select your wager:</label>
                   <br />
-                  <label>
-                    <input
-                      type='radio'
-                      onChange={(event) =>
-                        setWager(Number(event.currentTarget.value))
-                      }
-                      value={1}
-                      name='wager'
-                      checked={true}
-                    />
-                    $1
-                  </label>
-                  <input
-                    type='radio'
-                    onChange={(event) =>
-                      setWager(Number(event.currentTarget.value))
-                    }
-                    value={5}
-                    name='wager'
-                  />
-                  $5
+                  <Radio.Group
+                    onChange={(event: RadioChangeEvent) =>
+                      setWager(Number(event.target.value))
+                    }>
+                    <Radio style={radioStyle} value={1}>
+                      $1
+                    </Radio>
+                    <Radio style={radioStyle} value={5}>
+                      $5
+                    </Radio>
+                    <Radio style={radioStyle} value={10}>
+                      $10
```

Comment:
use `className` instead of hardcoded styles

Reply:
sure